lib/parseTriggers.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+var FirebaseError = require('./error');
+var fork = require('child_process').fork;
+var path = require('path');
+var RSVP = require('rsvp');
+
+var TRIGGER_PARSER = path.resolve(__dirname, './triggerParser.js');
+
+module.exports = function(projectId, sourceDir) {
```

Comment:
Is the only difference between this and the emulator trigger parser the `FIREBASE_PROJECT` env variable? If so, why not just have `firebaseConfig` be a third argument and don't use it to set `FIREBASE_PROJECT` if it's not there.